Resources/Prototypes/Maps/box.yml

```diff
@@ -2,7 +2,8 @@
   id: Box
   mapName: 'Box Station'
   mapPath: /Maps/box.yml
-  minPlayers: 50
+  minPlayers: 35
+  maxPlayers: 70
```

Comment:
is there a reason to set a max for box? it can definitely support 80 and i dont know how i feel about filtering out maps at 70+

Reply:
I think box, meta, and marathon can all support 80 players fine, this was mostly for pruning a few maps that people are bit tired of at the moment so that the high pop maps like Fland and Oasis are more likely to show up when the server is at max player count.